examples/torch/qat_after_ptq/resnet34/main.py

```diff
@@ -0,0 +1,325 @@
+    return downloader.get(DATASET_URL)
+
+
+def load_checkpoint(model: torch.nn.Module) -> torch.nn.Module:
+    checkpoint = torch.hub.load_state_dict_from_url(CHECKPOINT_URL, map_location=torch.device("cpu"), progress=False)
+    model.load_state_dict(checkpoint["state_dict"])
+    return model, checkpoint["acc1"]
+
+
+def main():
+    torch.manual_seed(0)
+    device = torch.device("cuda" if torch.cuda.is_available() else "cpu")
+    print(f"Using {device} device")
+
+    num_classes = 200  # 200 is for Tiny ImageNet, default is 1000 for ImageNet
+    init_lr = 1e-4
+
+    model = models.resnet18(pretrained=False)
+    # Update the last FC layer for Tiny ImageNet number of classes.
+    model.fc = nn.Linear(in_features=512, out_features=num_classes, bias=True)
+    model.to(device)
+
+    # Define loss function (criterion) and optimizer.
+    criterion = nn.CrossEntropyLoss().to(device)
+    optimizer = torch.optim.Adam(model.parameters(), lr=init_lr)
+
+    model, acc1_fp32 = load_checkpoint(model)
+    print(f"Accuracy of pre-trained FP32 model: {acc1_fp32:.3f}")
+
+    train_loader, val_loader, calibration_dataset = create_data_loaders()
+
+    def transform_fn(data_item):
+        return data_item[0].to(device)
+
+    quantization_dataset = nncf.Dataset(calibration_dataset, transform_fn)
+
+    quantized_model = nncf.quantize(model, quantization_dataset)
+    acc1_int8_init = validate(val_loader, quantized_model, criterion, device)
+    print(f"Accuracy of initialized INT8 model: {acc1_int8_init:.3f}")
+    print(f"Accuracy drop of initialized INT8 model over pre-trained FP32 model: {acc1_fp32 - acc1_int8_init:.3f}")
+
+    compression_lr = init_lr / 10
+    optimizer = torch.optim.Adam(quantized_model.parameters(), lr=compression_lr)
+
+    # Train for one epoch with NNCF.
+    train(train_loader, quantized_model, criterion, optimizer, epoch=0, device=device)
+
+    # Evaluate on validation set after Quantization-Aware Training (QAT case).
+    acc1_int8 = validate(val_loader, quantized_model, criterion, device)
+
+    print(f"Accuracy of tuned INT8 model: {acc1_int8:.3f}")
+    print(f"Accuracy drop of tuned INT8 model over pre-trained FP32 model: {acc1_fp32 - acc1_int8:.3f}")
+
+    input_shape = (1, 3, IMAGE_SIZE, IMAGE_SIZE)
+    example_input = torch.randn(*input_shape).cpu()
+
+    # Export FP32 model to OpenVINO™ IR
+    fp32_ir_path = f"{ROOT}/{BASE_MODEL_NAME}_fp32.xml"
+    ov_model = ov.convert_model(model.cpu(), example_input=example_input)
+    ov.save_model(ov_model, fp32_ir_path, compress_to_fp16=False)
+    print(f"FP32 model was exported to {fp32_ir_path}.")
+
+    # Export INT8 model to OpenVINO™ IR
+    int8_ir_path = f"{ROOT}/{BASE_MODEL_NAME}_int8.xml"
+    warnings.filterwarnings("ignore", category=TracerWarning)
+    warnings.filterwarnings("ignore", category=UserWarning)
+    ov_model = ov.convert_model(quantized_model.cpu(), example_input=example_input)
```

Comment:
nncf.strip()?

Reply:
I could not find any example with the `strip` call, could you please help me to find one?

Reply:
@alexsu52 `strip` is not needed for convert model  https://github.com/openvinotoolkit/nncf/pull/1988